tests/integration/api_network_test.py

```diff
@@ -323,13 +323,6 @@ def test_create_with_links(self):
 
         self.execute(container, ['nslookup', 'bar'])
 
-    def test_create_check_duplicate(self):
-        net_name, net_id = self.create_network()
-        with pytest.raises(docker.errors.APIError):
-            self.client.create_network(net_name, check_duplicate=True)
-        net_id = self.client.create_network(net_name, check_duplicate=False)
```

Comment:
alternatively, you could only remove the "=False" case; the other check in this test ("=True") should still be valid (but the "=True" will be a no-op in future)